doc/lsst.meas.base/generating_source_and_object_ids.rst

```diff
@@ -0,0 +1,26 @@
+.. py:currentmodule:: lsst.meas.base
+
+.. _lsst.meas.base-generating-source-and-object-ids:
+
+################################
+Generating source and object IDs
+################################
+
+The tables used to hold source and object measurements are expected to have row IDs that are unique across an entire processing run.
+This is achieved by first packing the :ref:`Butler data ID <lsst.daf.butler-dimensions_data_ids>` that identifies an in-memory table object into an integer, and then packing that with a per-table counter.
+The packing algorithms are reversible (they're not just a hash), and we always aim to fit the full row ID into a single 64-bit signed integer.
+In special cases, like LSST data releases, an identifier for the release may be packed in as well, making the row ID globally unique (for its type) across all releases.
+
+The `IdGenerator` class is the main entry point for generating these IDs.
+Usage starts with with defining an `lsst.pex.config` field for the type of data ID that identifies a table object:
+
+- `SkyMapIdGeneratorConfig` for ``{tract, patch}`` or ``{tract, patch, band}}`` data IDs (most object tables);
+- `DetectorVisitIdGeneratorConfig` for ``{visit, detector}`` data IDs (most source tables);
+- `DetectorExposureIdGeneratorConfig` for ``{exposure, detector}`` data IDs (rare, at least in the default LSST data model).
```

Comment:
I think these all need the full namespace in order to get built and linked.

Reply:
Oh, that's cool! I wasn't aware of that. I'll have to remember it for my own docs.